idd/Energy+.idd.in

```diff
@@ -43676,6 +43676,424 @@ Coil:Cooling:Water:DetailedGeometry,
        \note If blank or omitted, the Loop Design Temperature Difference value specified in Sizing:Plant
        \note object is used for sizing the Design Water Flow Rate.
 
+Coil:Cooling:DX,
+       \memo New better-than-bacon fantastic DX cooling coil.
+       \memo Includes DX evaporator coil, compressor, and condenser.
+       \min-fields 8
+   A1, \field Name
+       \required-field
+       \type alpha
+       \reference CoolingCoilsDX
+       \reference CoolingCoilsDXSingleSpeed
+       \reference CoolingCoilsDXMultiModeOrSingleSpeed
+       \reference CoolingCoilsDXVariableSpeed
+       \reference DesuperHeatingCoilSources
+       \reference AFNCoilNames
+   A2, \field Evaporator Inlet Node Name
+       \required-field
+       \type node
+   A3, \field Evaporator Outlet Node Name
+       \required-field
+       \type node
+   A4, \field Availability Schedule Name
+       \note Schedule value > 0 means the coil is available.
+       \note If this field is blank, the coil is always available.
+       \type object-list
+       \object-list ScheduleNames
+   A5, \field Condenser Zone Name
+       \note This input field is name of a conditioned or unconditioned zone where the secondary
+       \note coil (condenser) of DX system or a heat pump is to be placed.  This is an optional
+       \note input field specified only when user desires to reject the condenser heat into a
+       \note zone. The heat rejected is modelled as internal sensible heat gain of the zone.
+       \type object-list
+       \object-list ZoneNames
+   A6, \field Condenser Inlet Node Name
+       \type node
+   A7, \field Condenser Outlet Node Name
+       \type node
+   A8, \field Performance Object Name
+       \type object-list
+       \object-list DXCoolingPerformanceNames
+   A9, \field Condensate Collection Water Storage Tank Name
+       \type object-list
+       \object-list WaterStorageTankNames
+  A10, \field Evaporative Condenser Supply Water Storage Tank Name
+       \type object-list
+       \object-list WaterStorageTankNames
+  A11; \field Fuel Type
```

Comment:
@rraustad @Myoldmopar @nealkruis While writing transition code for the multispeed coil, I realized we didn't have a field for Fuel Type in the new coil. I've added it to the top level Coil:Cooling:DX but wondering if it should go in Coil:Cooling:DX:CurveFit:Performance instead.

Reply:
Wherever the energy use is calculated. I can't recall right now but I'm fairly sure it's higher than the speed level.